config/php-cs-fixer.php

```diff
@@ -2,154 +2,36 @@
-            '@PHP70Migration:risky' => true,
-            '@PHP71Migration' => true,
-            '@PHP71Migration:risky' => true,
+            '@PER-CS2.0' => true,
+            '@PER-CS2.0:risky' => true,
+
+            '@PHPUnit50Migration:risky' => true,
+            '@PHPUnit52Migration:risky' => true,
+            '@PHPUnit54Migration:risky' => true,
+            '@PHPUnit55Migration:risky' => true,
+            '@PHPUnit56Migration:risky' => true,
             '@PHPUnit57Migration:risky' => true,
             '@PHPUnit60Migration:risky' => true,
             '@PHPUnit75Migration:risky' => true,
             '@PHPUnit84Migration:risky' => true,
-            '@PER' => true,
-
-            // alias
-            'no_alias_functions' => true,
-
-            // array notation
-            'array_syntax' => ['syntax' => 'short'],
-            'no_trailing_comma_in_singleline' => true,
-            'no_whitespace_before_comma_in_array' => true,
-            'whitespace_after_comma_in_array' => true,
-
-            // basic
-            'encoding' => true,
-            'psr_autoloading' => true,
-
-            // casing
-            'magic_constant_casing' => true,
-            'native_function_casing' => true,
-
-            // cast notation
-            'cast_spaces' => ['space' => 'none'],
-            'lowercase_cast' => true,
-            'modernize_types_casting' => true,
-            'no_short_bool_cast' => true,
-            'short_scalar_cast' => true,
-
-            // class notation
-            'class_attributes_separation' => true,
-            'no_blank_lines_after_class_opening' => true,
-            'no_php4_constructor' => true,
-
-            // class usage
-            // (no rules used from this section)
-
-            // comment
-            'no_empty_comment' => true,
-            'single_line_comment_style' => true,
-
-            // constant notation
-            // (no rules used from this section)
-
-            // control structure
-            'elseif' => true,
-            'no_superfluous_elseif' => true,
-            'no_unneeded_control_parentheses' => true,
-            'no_unneeded_curly_braces' => true,
-            'no_useless_else' => true,
-            'trailing_comma_in_multiline' => true,
-            'yoda_style' => ['equal' => false, 'identical' => false, 'less_and_greater' => false],
-
-            // function notation
-            'type_declaration_spaces' => ['elements' => ['function', 'property']],
-            'native_function_invocation' => ['include' => ['@all']],
-            'no_spaces_after_function_name' => true,
-            'return_type_declaration' => ['space_before' => 'none'],
 
-            // import
-            'no_leading_import_slash' => true,
-            'no_unused_imports' => true,
-            'ordered_imports' => true,
+            // overwrite the PER2 defaults to restore compatibility with PHP 7.x
+            'trailing_comma_in_multiline' => ['elements' => ['arrays', 'match']],
```

Comment:
As per issue on sister project, we can't include `match` here, until PHP-CS-Fixer fixes the issue.

Reply:
It's removed now.

Reply:
Oh, it seems `match` was there 2 times. I've removed the other one now as well, and also created https://github.com/MyIntervals/PHP-CSS-Parser/pull/629.